scripts/linters/js_ts_linter.py

```diff
@@ -56,6 +56,20 @@
 _MESSAGE_TYPE_SUCCESS = 'SUCCESS'
 _MESSAGE_TYPE_FAILED = 'FAILED'
 
+_BANNED_ESLINT_DISABLE_STATEMENTS = [
+    {
+        'pattern': r'eslint-(disable|enable)(-next-line)? camelcase',
+        'rule_name': 'camelcase',
+        'custom_message': (
+            'If you are using this statement to define properties '
+            'in an interface for a backend dict. Wrap the property '
+            'name in single quotes instead.'),
+        'exceptions': [
+            'typings/guppy-defs-b5055b963fdbea5c6c1e92dbf58fdaf3ea0cd8ba.d.ts',
+            'core/templates/services/UpgradedServices.ts']
```

Comment:
These were needed because these files had variables in `snake_case` that couldn't use quotes.